scss/layout/_scaffolding.scss

```diff
@@ -0,0 +1,122 @@
+@import '../abstracts/variables';
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+[class*='container'] {
+  margin: 0 auto;
+  padding-right: $column-gutter;
+  padding-left: $column-gutter;
+}
+.container-small {
+  max-width: $device-sm;
+}
+.container {
+  position: relative;
+  max-width: 66rem;
+}
+.container-large {
```

Comment:
Do we currently have references to these container names already in `webb3`? I would be scared this might break something if we already use these class names.